services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelperV2.groovy

```diff
@@ -141,14 +135,7 @@ class FlowHelperV2 {
         log.debug("Adding flow '${flow.flowId}'")
         def response = northboundV2.addFlow(flow)
 
-        def flowEntry = null
-        Wrappers.wait(WAIT_OFFSET) {
-            assert northbound.getFlowStatus(flow.flowId).status == FlowState.UP
-
-            flowEntry = db.getFlow(flow.flowId)
-            assert flowEntry
-        }
-        checkRulesOnSwitches(flowEntry, RULES_INSTALLATION_TIME, true)
+        Wrappers.wait(WAIT_OFFSET) { assert northbound.getFlowStatus(flow.flowId).status == FlowState.UP }
```

Comment:
not enough to just remove `checkRulesOnSwitches`. There should be separate test/verification for v2 api which ensures that 'UP' status is set when and only when all the rules are actually set on all the switches.

Reply:
fixed
new test was added `def "System marks flow as UP when and only when all the rules are actually set on all involved switches"() `